src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -14,16 +14,22 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<configurator>
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/service_device_management/OperationalCredentialCluster.adoc
+Parameters: in-progress 
+Git: 0.7-summer-2025-dirty
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
   <domain name="CHIP"/>
-
   <struct name="FabricDescriptorStruct" isFabricScoped="true">
     <cluster code="0x003E"/>
-    <item fieldId="1" name="RootPublicKey" type="octet_string" length="65"/>
+    <item fieldId="1" name="RootPublicKey" type="octet_string" length="65" minLength="65"/>
     <item fieldId="2" name="VendorID" type="vendor_id"/>
     <item fieldId="3" name="FabricID" type="fabric_id"/>
     <item fieldId="4" name="NodeID" type="node_id"/>
     <item fieldId="5" name="Label" type="char_string" length="32"/>
+    <item fieldId="6" name="VidVerificationStatement" type="octet_string" optional="true" length="85" minLength="85"/>
```

Comment:
This has nothing to do with cameras....

Furthermore, shouldn't there be provisional annotations on the VID verification bits being added here and elsewhere in this file?

Reply:
Fun, apologies, not sure why alchemy re-generated this file when I didn't explicitly ask it to (it normally updates the globals but not the leaves w/o an explicit cli arg)